src/app/(auth)/sign-in/page.tsx

```
@@ -1,10 +1,20 @@
 import { SignInView } from "@/modules/auth/ui/views/sign-in-view";
 
-const Page = () => {
-  return (
-    <SignInView />
-  )
+const Page = async () => {
+  const session = await auth.api.getSession({
+    headers: await headers(),
+  });
+
+  if (!!session) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)